manifest.json

```diff
@@ -2097,6 +2175,16 @@
                 }
             },
             "versions": {
+                "1.0.2": {
+                    "releaseUrl": "https://github.com/rassi0429/NeosPasteTweak/releases/tag/v1.0.2",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/rassi0429/NeosPasteTweak/releases/download/v1.0.2/NeosPasteTweak.dll",
+                            "filename": "NeosPasteTweak.dll",
+                            "sha256": "76862cd08e9330ccf229aed2f47c2ba42a7ea5ca9e4256c97c4fcf77cb1c8a2f"
+                        }
+                    ]
+                },
```

Comment:
Very sussy huge blocks of JSON text hardcoded (should probably be done with [assembly embedded recourses](https://github.com/SleepyVRC/Mods/blob/main/PrivateInstanceIcon/PrivateInstanceIcon.csproj#L16=) ([usage](https://github.com/SleepyVRC/Mods/blob/main/PrivateInstanceIcon/PrivateInstanceIconMod.cs#L47=)))

Reply:
Would be more probably more ideal. or using a neosdb url to import as they are essentially just spawning an existing item then set the text inside of when spawning it.

Reply:
Looks like those large blocks did get moved into a different cs file atleast in the repo, no build/release made with that change yet.